providers/aws/config.yaml

```diff
@@ -0,0 +1 @@
+shodan_api_key: null
```

Comment:
Why `null` ?

I think for the checks we need to use the config file instead of command line arguments. It's better to only have on path for that kind of operation.

Reply:
If not it would be a little mess trying to find what value is the right one.

Reply:
I think that we have to have the same functionality as version 2, at least at the beginning. That's why the best option for keeping both feature is to do it this way.